connaisseur/validators/cosign/cosign_validator.py

```diff
@@ -1,4 +1,3 @@
-# import asyncio
```

Comment:
I believe my fix was released and consequently, the asyncio parallelization might work now. So one might check, whether it's fast or less consuming. Bit hesitant to simply remove it

Reply:
Well, either we want to have and use it or not. We shouldn't have random commented out code. If we want it back, we can simply git checkout an old version

Reply:
well, ok, than it is gone. fine